packages/menu/src/Menu.spec.js

```diff
@@ -71,7 +71,12 @@ describe('packages/Menu', () => {
         >
           Item 1
         </MenuItem>
-        <MenuItem href="http://mongodb.design" data-testid="second-item">
+        <MenuItem
+          href="http://mongodb.design"
+          data-testid="second-item"
+          target="_self"
+          rel=""
```

Comment:
Should probably check that an actual value is carried through. Empty string may be a false positive.

Reply:
Updated!